routers/private/hook_post_receive.go

```diff
@@ -208,7 +208,7 @@ func HookPostReceive(ctx *gitea_context.PrivateContext) {
 			return
 		}
 
-		cols := make([]string, 0, len(opts.GitPushOptions))
+		cols := make([]string, 0, 2)
```

Comment:
why not using len of push options?

Reply:
But why using len of push options?

Reply:
> But why using len of push options?

I think it's better set a length when we know the max length of a slice.

Reply:
But there could be far more push options than it really needs here.

At most, here it only needs 2, but there could be 4 or 5 or even more push options. They are not related.